### DISCUSSION TITLE
AI 101

### DISCUSSION THREAD

**Discussion by TinyHust** [TinyHust/notes#437] [AI - ML - DL - CS - DS]
## LLM
- [llamaindex](https://www.llamaindex.ai/)
- [langchain](https://www.langchain.com/)
  - [examples](https://github.com/TinyHust/notes/discussions/413)
  - LangGraph
- local LLM 
  - [lmstudio](https://lmstudio.ai/)
  - [vllm ](https://github.com/vllm-project/vllm)
  - [ollama](https://ollama.com/)
    - [open-webui](https://github.com/open-webui/open-webui)
- [dynamo](https://github.com/ai-dynamo/dynamo)
- [lmstudio](https://lmstudio.ai/)
- [jan](https://jan.ai/)
- [cline](https://github.com/cline/cline)

**Comment by mediumhust** [comment]
### Text-to-SQL
- https://docs.llamaindex.ai/en/stable/examples/index_structs/struct_indices/SQLIndexDemo/
- https://colab.research.google.com/drive/1cCbobjnqqGfiBWmBQmIa9p4Ik_wVR1on
- https://github.com/mediumhust/langchain/blob/main/4_sqldb_tshirts/t_shirt_sales_llm.ipynb

### Ollama
- [chatbot](https://github.com/mediumhust/ollama-chatbot)
- https://github.com/mediumhust/Langchain-and-Ollama
- https://github.com/mediumhust/LangGraph-and-Ollama

### document processing
- [docling](https://github.com/DS4SD/docling)
  - doc to pdf

**Comment by mediumhust** [comment]
## MCP
- [DesktopCommanderMCP](https://github.com/wonderwhy-er/DesktopCommanderMCP)
- [task-management](https://github.com/eyaltoledano/claude-task-master)

**Reply by mediumhust** [reply]
### MCP Server
- [AWS MCP Server](https://github.com/awslabs/mcp)
- [GCP MCP Server](https://github.com/eniayomi/gcp-mcp)
- [Microsoft MCP](https://github.com/microsoft/mcp)
  - [Azure MCP Server](https://github.com/Azure/azure-mcp)
- [Cloudflare MCP Server](https://github.com/cloudflare/mcp-server-cloudflare)
- [Docker MCP Server](https://github.com/QuantGeekDev/docker-mcp)
- [Kubernetes MCP Server](https://github.com/Flux159/mcp-server-kubernetes)
- [Github MCP Server](https://github.com/github/github-mcp-server)
- [Git MCP Server](https://github.com/modelcontextprotocol/servers/tree/main/src/git)
- [Azure Devops Server](https://github.com/Tiberriver256/mcp-server-azure-devops)
- [PostgreSQL MCP Server](https://github.com/crystaldba/postgres-mcp)
- [MySQL MCP Server](https://github.com/benborla/mcp-server-mysql)
- [MongoDB MCP Server](https://github.com/mongodb-js/mongodb-mcp-server)
- [Terraform MCP Server](https://github.com/hashicorp/terraform-mcp-server)
- [Pulumi MCP Server](https://www.pulumi.com/docs/iac/guides/ai-integration/mcp-server/)

**Comment by mediumhust** [comment]
## Libs
- [diffusers](https://huggingface.co/docs/diffusers/index)
  - checkpoint
    - [civitai](https://civitai.com/)
    - https://huggingface.co/models?library=diffusers
  - [ComfyUI](https://github.com/comfyanonymous/ComfyUI)
    - [examples](https://comfyanonymous.github.io/ComfyUI_examples/)
    - [InstantID](https://github.com/cubiq/ComfyUI_InstantID)
    - [ControlNet](https://huggingface.co/xinsir/controlnet-union-sdxl-1.0)
    - [LivePortraitKJ](https://github.com/kijai/ComfyUI-LivePortraitKJ)

**Comment by mediumhust** [comment]
### RAG
- [superlinked](https://github.com/superlinked/superlinked)
- pgvector
  - [pgvector](https://github.com/pgvector/pgvector)
  - https://www.crunchydata.com/blog/smarter-postgres-llm-with-retrieval-augmented-generation

**Comment by mediumhust** [comment]
### Workflow
- [AutoGPT](https://github.com/Significant-Gravitas/AutoGPT)
- [langflow](https://github.com/langflow-ai/langflow)
- [crewAI](https://github.com/crewAIInc/crewAI)
- [autogen](https://github.com/microsoft/autogen)

**Comment by mediumhust** [comment]
### Video
- [Wan2GP](https://github.com/deepbeepmeep/Wan2GP)
  - [Wan2.1](https://github.com/Wan-Video/Wan2.1)
- [ltx](https://ltx.io/model)

**Comment by mediumhust** [comment]
### Prompt
- https://opennana.com/

**Comment by mediumhust** [comment]
## Agent
- [pixel-agents](https://github.com/pablodelucca/pixel-agents)

**Comment by mediumhust** [comment]
## Skill
- [caveman](https://github.com/juliusbrussee/caveman)
- [antigravity-awesome-skills](https://github.com/sickn33/antigravity-awesome-skills)

**Comment by mediumhust** [comment]
### Ref:
- [QuantDinger](https://github.com/brokermr810/QuantDinger)
- [browser-harness](https://github.com/browser-use/browser-harness)
- https://github.com/anthropics/financial-services
- [ruflo](https://github.com/ruvnet/ruflo)
